tests/hp/fe_nothing_25.output

```diff
@@ -0,0 +1,22 @@
+
+DEAL:1d::constraints:
+DEAL:1d::dof values:
```

Comment:
So this isn't correct, actually. You'd expect that one of the DoFs on one of the cells is actually constrained.